sdks/python/tox.ini

```diff
@@ -27,6 +27,8 @@ select = E3
 
 # Shared environment options.
 [testenv]
+# allow apps that support color to use it.
+passenv=TERM
```

Comment:
Now you can see mypy color output from tox!  Let me know if you want me to do this in another PR.